### DISCUSSION TITLE
"Single Midi file player Operator " Request

### DISCUSSION THREAD

**Discussion by jimimased** [cables-gl/cables_docs#365] [Feature Requests] [answered]
I'd like to deploy a patch I've made as totally self contained to the browser online, however the vision is triggered by 4 midi files ( Im running on ableton).  
A simple single channel "Midi file player Operator" would be a most useful addition, I could then load the 4 midi files into the patch  & deploy the patch online in the browser with ease. There may be alternative solution around this issue Im not aware of (Id like to know what solutions there are currently) but a "Midi file player Operator" would prove very useful. Thanks very much for your great work!

**Comment by steam0r** [comment, marked as answer]
hey,

can't you use Ops.Audio.MidiJson and Ops.Json.SwitchObject to get to what you need? When exporting the patch make sure to select the "all" option for "include assets".

best,
stephan

**Reply by jimimased** [reply]
Thank you Stephan!
